src/components/statemeta.js

```diff
@@ -129,9 +141,13 @@ function StateMeta({
             '(((previousDayData - sevenDayBeforeData) / sevenDayBeforeData) * 100)/7'
           }
           date={`${sevenDayBeforeDate} - ${previousDayDate}`}
-          description={`In the last one week, the number of new infections has grown by an average of ${Math.round(
-            growthRate / 7
-          )}% every day.`}
+          description={
+            growthRate != 0
+              ? `In the last one week, the number of new infections has grown by an average of ${Math.round(
+                  growthRate / 7
+                )}% every day.`
+              : `There has been no growth in the number of cases in last 7 days.`
```

Comment:
Lets keep this in sync with the other message? Replace 7 days with last week, cases with infections?

Reply:
Sure, added a new commit with the change.